docs/document/content/user-manual/shardingsphere-scaling/usage.cn.md

```diff
@@ -134,11 +156,11 @@ GENERATED_KEY(COLUMN=order_id,TYPE(NAME=snowflake,PROPERTIES("worker-id"=123)))
 );
 ```
 
-**注意**：当前版本不支持通过修改`TableRule`触发迁移。
+**注意**：当前版本不支持通过修改`TableRule`触发迁移。对于同一个逻辑表，不能混合使用这2种格式。
 
 #### 查询所有迁移任务
 
-详情请参见[RAL#弹性伸缩](/cn/user-manual/shardingsphere-proxy/usage/distsql/syntax/ral/ral/#%E5%BC%B9%E6%80%A7%E4%BC%B8%E7%BC%A9)。
+详情请参见[RAL#弹性伸缩](/cn/user-manual/shardingsphere-proxy/usage/distsql/syntax/ral/#弹性伸缩)。
```

Comment:
It's not recommend to use Chinese word in http url.

Reply:
ok, I will change this